sphinx/builders/epub3.py

```diff
@@ -275,7 +308,7 @@ def setup(app):
     app.add_builder(Epub3Builder)
     app.connect('builder-inited', validate_config_values)
 
-    app.add_config_value('epub_description', '', 'epub3', string_classes)
+    app.add_config_value('epub_description', 'unknown', 'epub3', string_classes)
```

Comment:
Other settings of epub use same default value (``unknown``):

* ``epub_contributor``
* ``epub_author``
* ``epub_publisher``
* ``epub_identifier``
* ``epub_scheme``
* ``epub_uid``